src/FffGcodeWriter.h

```diff
@@ -715,9 +715,23 @@ class FffGcodeWriter : public NoCopy
      */
     unsigned int findSpiralizedLayerSeamVertexIndex(const SliceDataStorage& storage, const SliceMeshStorage& mesh, const int layer_nr, const int last_layer_nr);
 
-    static bool partitionInfillBySkinAbove(Polygons& infill_below_skin, Polygons& infill_not_below_skin,
-                                           const LayerPlan& gcode_layer, const SliceMeshStorage& mesh,
-                                           const SliceLayerPart& part, coord_t infill_line_width) ;
+    /*!
+     * Partition the Infill regions by the skin at N layers above.
+     *
+     * When skind edge support layers is set this function will check N layers above the infill layer to see if there is
```

Comment:
'skind'?

Reply:
Yes @rburema, keep up with the Cura street lingo. Al the cool kids are saying skind instead of skin.

But I will change it just for you ;-)

Reply:
I think that's absur. Very absur.